Gemfile

```diff
@@ -8,6 +8,7 @@ source "https://rubygems.org"
 #
 # To upgrade, run `bundle update github-pages`.
 gem "github-pages", group: :jekyll_plugins
+gem "webrick"
```

Comment:
What is this new package for? Can you please revert this change.

Reply:
The package is necessary for me to run Jekyll, but I'll find another workaround.